all-repos.yaml

```diff
@@ -122,3 +122,4 @@
 - https://github.com/talos-systems/conform
 - https://github.com/twu/skjold
 - https://github.com/woile/commitizen
+- https://github.com/hcodes/yaspeller.git
```

Comment:
none of the others end with .git -- why did you do it for this one?

Reply:
It was giving me a failure when I tested it , I can remove it and leave like the others

Reply:
looks good but have to resolve the conflicts

Reply:
done!